### ISSUE TITLE
Output table for summary(reduced.form)

### ISSUE THREAD

**Issue by MislavSag** [alexanderlange53/svars#80]
Is there a function in your or some other package that converts `summary(reduced.form)` to a nice regression table? Something similar to gt_summary for lm object. I know it is possible to construct table 'mannualy', but it would be helpfull if there is already a solution for this.

**Comment by BDalheimer** [comment]
I agree that this would be very useful. We are thinking about including some markdown/latex compatible solutions for the structural output. In that process it's straightforward to also implement a function for the reduced forms, even though these models are not estimated by ```svars``` but by other packages, e.g. `vars`. Are you thinking about side-by-side type regression output for the individual reduced form equations?
